pages/common/git-diff-tree.md

```diff
@@ -0,0 +1,20 @@
+# git diff-tree
+
+> Compares the content and mode of blobs found via two tree objects.
+> More information: <https://git-scm.com/docs/git-diff-tree>.
+
+- Compare two tree objects:
+
+`git diff-tree {{tree-ish}} {{tree-ish}}`
+
+- Show changes between two specific commits:
+
+`git diff-tree -r {{commit}} {{commit}}`
+
+- Display changes in patch format:
+
+`git diff-tree -p {{tree-ish}} {{tree-ish}}`
+
+- Filter changes by a specific path:
+
+`git diff-tree {{tree-ish}} {{tree-ish}} -- {{path/to/file_or_directory}}`
```

Comment:
```suggestion
`git diff-tree {{tree-ish1}} {{tree-ish2}}`

- Show changes between two specific commits:

`git diff-tree -r {{commit1}} {{commit2}}`

- Display changes in patch format:

`git diff-tree -p {{tree-ish1}} {{tree-ish2}}`

- Filter changes by a specific path:

`git diff-tree {{tree-ish1}} {{tree-ish2}} -- {{path/to/file_or_directory}}`
```
A minor suggestion to distinguish the placeholders.

Reply:
oh, right! thank you